.docker/qgis.dockerfile

```
@@ -8,6 +8,9 @@ ARG CACHE_DIR
 FROM  qgis/qgis3-build-deps:${DOCKER_TAG}
 MAINTAINER Denis Rouzaud <denis@opengis.ch>
 
+LABEL Description="Docker container with QGIS for CI testing" Vendor="QGIS.org" Version="1.1"
```

```suggestion
LABEL Description="Docker container with QGIS" Vendor="QGIS.org" Version="1.1"
```

there is also server, etc. It is not aimed at CI testing only.